src/main/java/de/sakpaas/backend/service/SearchService.java

```diff
@@ -193,15 +190,26 @@ protected SearchRequest getByCoordinates(SearchRequest request) {
       locations = locations.stream()
           .filter(location -> {
             for (String brand : knownBrands) {
```

Comment:
Ist hier nicht allgemein noch ein Denkfehler drin? Weil du checkst ja, ob der Brand der Location in knownBrands drinne ist. Jedoch müssen wir doch nur prüfen ob ein Brand aus request.getBrand() vorkommt.

In knwonBrands sind ja alle Brands drinne.

Reply:
![image](https://user-images.githubusercontent.com/52504463/81003027-7c28e380-8e4a-11ea-9803-d5fb84f935a9.png)
Ist hier mit `$Brands[]` `knownBrands` gemeint?

Reply:
ja